src/messaging/ExchangeMgr.cpp

```diff
@@ -191,11 +191,38 @@ void ExchangeManager::OnMessageReceived(const PacketHeader & packetHeader, const
 {
     UnsolicitedMessageHandlerSlot * matchingUMH = nullptr;
 
+    auto * protocolName = Protocols::GetProtocolName(payloadHeader.GetProtocolID());
```

Comment:
Is it possible to wrap this string processing behind an #ifdef so really lean-and-mean production builds can disable human readable logs?